Form-Controls/index.html

```diff
@@ -13,15 +13,70 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+        <div>
+          <label for="fname">First name</label>
+          <input type="text" name="fname" id="fname" required>
+        </div>
+      </br>
+        <div>
+          <label for="lname">Last name</label>
+          <input type="text" name="lname" id="lname" required>
+        </div>
+      </br>
+        <div>
+          <label for="email">Email</label>
+          <input type="email" name="email" id="email" required>
+        </div>
+        <div>
+          <h3>What colour of a t-shirt do you want?</h3>
+        <div>
+          <label for="colour">Purple</label>
```

Comment:
You are missing connection between `label `and corresponding `input.` Label has `for` attribute for this purpose. Which attribute should be used in `input `in this case? It applies to all your labels and inputs.